part-08.mdx

```diff
@@ -0,0 +1,230 @@
+# Build your first custom dashboard
+
+In previous parts of the tutorial, you have learned how several sections of Netdata dashboard worked. 
+
+This guide will show you how to easily create custom pages and set up a custom dashboard to fit your custom 
+or specific needs.
```

Comment:
Can probably cut "...custom or..." from this, since custom dashboards implies custom applications.